Marlin/fonts/README.fonts

```diff
@@ -1,4 +1,4 @@
-The fonts are created with Fony.exe (http://hukka.ncn.fi/?fony) because Fontforge didn't do what I want (probably lack off experience). 
+The fonts are created with Fony.exe (http://hukka.ncn.fi/?fony) because Fontforge didn't do what I want (probably lack of experience).
```

Comment:
@Wackerbarth 
Thank you for correcting my spelling errors - but this has nothing to do with the formatting.

@thinkyhead 
The same in your #2668 - but at least announced.
By the way - in qr-solve.cpp with the new formatting it's obvious that we can replace at least 3 of the functions with the standard macros (min(), max(), abs() and probably also sign())